quartz_solar_forecast/utils/sentry_logging.py

```diff
@@ -14,15 +12,22 @@
 )
 
 SENTRY_DSN = "https://b2b6f3c97299f81464bc16ad0d516d0b@o400768.ingest.us.sentry.io/4508439933157376"
-sentry_sdk.init(dsn=SENTRY_DSN, traces_sample_rate=1.0)
+
+sentry_sdk = None
+# Only import/init the sentry_sdk if the environment has not opted out of logging
+# see https://github.com/openclimatefix/open-source-quartz-solar-forecast/issues/317
+if quartz_solar_forecast_logging:
+    import sentry_sdk
+    sentry_sdk.init(dsn=SENTRY_DSN, traces_sample_rate=1.0)
 
 
 def write_sentry(params):
     """
     Log usage of this package to Sentry
     """
 
-    if not quartz_solar_forecast_logging:
+    # Skip logging if not imported and initialized
+    if sentry_sdk is None:
```

Comment:
im not sure this works, sentry_sdk would not be defined, 
Could you move it back to ` if not quartz_solar_forecast_logging:`